modules/exploits/linux/http/f5_icontrol_rce.rb

```diff
@@ -0,0 +1,166 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'F5 BIG-IP iControl REST authentication bypass',
+        'Description' => %q{
+          This module exploits a authentication bypass vulnerability
+          in the F5 iControl REST service to execute commands on an
+          affected BIG-IP trough the endpoint `/mgmt/tm/util/bash`.
+        },
+        'Author' => [
+          'Heyder Andrade', # Metasploit module
+          'alt3kx <alt3kx[at]protonmail.com>' # PoC
+        ],
+        'References' => [
```

Comment:
This still needs to link to the PoC.

Reply:
I assume this is https://github.com/alt3kx/CVE-2022-1388_PoC, will update this accordingly now.

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/16549/commits/61f5303fc43631c834dcbe81cdc8b2ddf700a6c8